Spamblock plugin / last multilingual bugfix

stephanbrunker opened the pull request:
I have divided it into two commits: One is the fix for the spamblock plugin: It now uses a timeout between showing the page for reading and allowing contents. You can use a fixed amount of time or an adaptive one, depending on reading and writing speed. That is tested and works, now I don't have any spamcomments anymore, but just got a real one with the adaptive setting to 500 words/chars per minute. To circumvent that, the spammers need to keep open each connection for minutes and that probably slows down their operations a bit because they hopefully can't have that big a number of connections parallel. 

The second one is changing the names of the languages to their own languages as was requested in the forum. And fixing the last remaining language bug which was the initialition point of the plugins -> Issue #657 

That second one should be tested by someone who has openid plugin installed. I don't have and don't have openid able accounts, but it should work anyway.

onli commented:
Ok, I'll merge this now. Please in the future try to make one dedicated PR for each change. For example the nl2br change does not belong to this. Probably you just forgot to switch back to the default/a different branch after making the PR?

th-h commented:
Yes, it's really a bit unfortunate to mix three unrelated changes in this way. :-/

I'll try (!) to document them accordingly in NEWS/Changelog (and hope I understand what has been done ...)

stephanbrunker commented:
Sorry, I have no experience with project workflow if you work in a team and make it up on the seat of my pants ;-)

Upside is: I can only learn from everything I do. In this case: creating several forks in my git and merging them only after the PR is merged upstream. For the first PR, i added everything to the changelog, I just forgot to do that with the second one. For the additional plugins, I added everything into the changelog files.

th-h commented:
No problem, and sorry for sounding a bit grumpy. :)

stephanbrunker commented:
Also no problem, I made changes in a lot of areas … that makes it complicated.
